mmcv/runner/epoch_based_runner.py

```diff
@@ -67,6 +67,7 @@ def val(self, data_loader, **kwargs):
             self.run_iter(data_batch, train_mode=False)
             self.call_hook('after_val_iter')
 
+        self._iter += 1
```

Comment:
Thanks for your feedback.
Maybe increasing the self._iter is not a proper solution because lr scheduler updater hook depend on it.

https://github.com/open-mmlab/mmcv/blob/03491ed28a6f07c766b7a6950452b1782d95dae4/mmcv/runner/hooks/lr_updater.py#L124-L132

Reply:
Well, so we can't modify self._iter in val func. Then, how about just delete step param in log func of wandb logger? like this:
In mmcv/runner/hooks/logger/wandb.py, log func of WandbLoggerHook class. from:
```python
    @master_only
    def log(self, runner):
        tags = self.get_loggable_tags(runner)
        if tags:
            self.wandb.log(
                tags, step=self.get_iter(runner), commit=self.commit)
```
change to
```python
    @master_only
    def log(self, runner):
        tags = self.get_loggable_tags(runner)
        if tags:
            self.wandb.log(
                tags, commit=self.commit)
```

Reply:
Approve.

Reply:
ok，I will close this pr and create a new pr.